examples/quick_start_examples/user_profiles/connector.py

```diff
@@ -86,6 +91,42 @@ def update(configuration: dict, state: dict):
     state["login_cursor"] = cursor
     yield op.checkpoint(state)
 
+    # Upsert tables with NaN
+    cursor_2 = state["table_with_nan_cursor"] if "table_with_nan_cursor" in state else 0
+    table_df_1 = generate_data_with_NaN()
+    table_df_2 = generate_data_with_NaN()
+    table_df_3 = generate_data_with_NaN()
+
+    # Approaches to replace NaN with None
+
+    # Convert NaN to None using the replace method
+    table_df_1 = table_df_1.replace(np.nan, None)
```

Comment:
I think we should create a method and add proper comments like
Approach 1:
Approach 2:
etc